prestoadmin/server.py

```diff
@@ -125,6 +127,15 @@ def uninstall():
         print('Package uninstalled successfully on: ' + env.host)
 
 
+@task
+@requires_topology
+def upgrade(local_package_path, local_config_dir):
```

Comment:
Tasks all need to have pydoc like in configuration deploy, because it's user facing (when you do ./presto-admin -d configuration deploy, for example, it prints out the pydoc)